packages/trader/src/AppV2/Components/PayoutInfo/__tests__/payout-info.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import PayoutInfo from '../payout-info';
+import { TContractInfo } from '@deriv/shared';
+
+describe('PayoutInfo component', () => {
+    const mockContractInfo: TContractInfo = {
+        longcode: 'This is a test longcode for the contract.',
+    };
+
+    it('renders without crashing', () => {
+        render(<PayoutInfo contract_info={mockContractInfo} />);
+        expect(screen.getByText('How do I earn a payout?')).toBeInTheDocument();
```

Comment:
Maybe it'll be better to create a constant with this reusable string?

Reply:
Again why is it needed? It's a test file.